### PR TITLE
Docs/windows watch

### PR DESCRIPTION
**Pull request by Alysha-94**
Add some documentation for `watchexec` as an alternative for the `watch` command for windows

### CODE REVIEW

**Comment by shuttle1987** [comment]
Thanks for this!

File: docs/FAQ.rst [hunk 1]
```diff
@@ -31,4 +32,37 @@ This will be because you have a version of setuptools that is too old, Gilbert r
...
 You can fix this by updating versions:
 
-    pip install -U setuptools
\ No newline at end of file
+.. code-block:: sh
+
+    pip install -U setuptools
+	
+How do I have Gilbert automatically rebuild the site when files change?
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```
Comment (funkybob) [inline docs/FAQ.rst]: My personal preference is to have the header strings match the length of the header.

File: docs/FAQ.rst [hunk 2]
```diff
@@ -31,4 +32,37 @@ This will be because you have a version of setuptools that is too old, Gilbert r
...
+
+Have gilbert watch your files, and re-render on changes:
```
Comment (funkybob) [inline docs/FAQ.rst]: Can we rearrange this to say "If you have Linux, you can use the build in watch command" sort of approach, and propose watchexec for every other OS [MacOS, Windows, even Linux too :) ]

**Review by funkybob — CHANGES_REQUESTED** [review]
Thanks for this! Just a couple of small changes before we merge it in, ok?

File: docs/FAQ.rst (hunk 2 above)
Comment (Alysha-94) [reply docs/FAQ.rst]: Sure, thanks for the feedback! I originally thought that it was windows specific, so I just updated it to be more general for every other OS

**Comment by funkybob** [comment]
Excellent. Thanks!

**Pull request merged**